source/robot_model/src/Model.cpp

```diff
@@ -150,6 +150,45 @@ state_representation::Jacobian Model::compute_jacobian(const state_representatio
   return this->compute_jacobian(joint_positions, frame_id);
 }
 
+Eigen::MatrixXd Model::compute_jacobian_time_derivative(const state_representation::JointPositions& joint_positions,
+                                                        const state_representation::JointVelocities& joint_velocities,
+                                                        unsigned int frame_id) {
+  if (joint_positions.get_size() != this->get_number_of_joints()) {
```

Comment:
Shouldn't we check the size of the joint_velocities too?

Reply:
Right!